package.json

```diff
@@ -3,7 +3,16 @@
   "version": "1.2.0",
   "description": "A parser for the Java/Maven pom.xml files.",
   "license": "MIT",
-  "main": "lib/",
+  "source": "lib/index.js",
+  "main": "dist/index.js",
+  "umd:main": "dist/index.js",
```

Comment:
Please use `index.umd.js` as the filename.

Reply:
done